apps/s_server.c

```diff
@@ -3450,7 +3487,12 @@ static int rev_body(int s, int stype, int prot, unsigned char *context)
     BIO_push(io, ssl_bio);
     ssl_bio = NULL;
 #ifdef CHARSET_EBCDIC
-    io = BIO_push(BIO_new(BIO_f_ebcdic_filter()), io);
+    BIO *filter;
```

Comment:
This declaration has to be done at the beginning of the block and needs to be guarded by the `#ifdef` there as well.

Reply:
You've forgot to drop the declaration here.